src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -5123,6 +5166,41 @@ components:
             type: string
             example: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJnYTRnaF9wYXNzcG9ydF92MSI6W119.JJ5rN0ktP0qwyZmIPpxmF_p7JsxAZH6L6brUxtad3CM
           description: the encoded JWT GA4GH Passport that contains embedded Visas.  The overall JWT is signed as are the individual Passport Visas.
+    DRSAuthorizations:
```

Comment:
Noted some inconsistent casing so was on the fence re: `DRS` vs. `Drs`, but please let me know if there's one we prefer.

Reply:
Oh yeah, good catch, that's unfortunate. Given most of these other model names are "DRS", I think sticking with that in this case makes sense.